hyahatiph-labs/gitpayd
# Reemuru/bug fixes

Closes #20 

### Description

Update log level for filtering
added code of conduct and contributing docs
### Payment Request

LN:lnbc1u1psx603lpp5fl44ytn0mfdzy8nh2tkl5c3kn79hphaqtc7sknuh0dum4m4nzwjsdqdg9jxgg8sn7f27cqzpgxqr23ssp5htw6grmx7y82ptgufvhjyetpr6qx7ymsfhp4720emq4y76jhav8q9qyyssquvy2rluxsaf73hnqq3w4cqdd628azjxdtj24xz82t05erqmr5uv4wfqf4a3m5fkyk4ejp4vvjcw978u5uenp832hx0y77q47rpk8jegqj69f4d

This repo is powered by the gitpay daemon and LND. Get paid for closing issues
<br/>
Learn more about LND at https://dev.lightning.community/

### Steps to Test

1. Execute workflow with action
2. verify gitpayd  succeeded and sent payment

### Screenshots

[Only if applicable]
